editor/src/clj/editor/tile_map.clj

```diff
@@ -179,6 +179,8 @@
 
 (def erase-brush (make-brush nil))
 
+(def ^:private empty-brush {:width 1 :height 1 :tiles [{:tile nil :h-flip false :v-flip false :rotate90 false}]})
+
```

Comment:
It looks like we could also use `(make-brush nil)` here? 🤔

Reply:
I mentioned it in comment above:
>we use it in render, that's why I don't just use make brush to avoid new brush creation every frame

but maybe I don't understand right how it works. isn't this way just a single table which exist all the time?

Reply:
`def` happens only once, when the file is loaded. Its similar to having
```
local empty_brush = make_brush(nil)
```
Outside of any of the functions in Lua.

Reply:
Then I misunderstood you
I thought you wanted me to use `(make-brush nil)` everywhere where I use empty-brush. I'll fix it. Thank you